GitHub.Collectors.Functions/Settings.barebones.template.json

```diff
@@ -22,5 +22,6 @@
         "TrafficTimer": {},
 
         "Traffic": {}
-    }
+    },
+    "ApiDomain":  "api.github.com"
```

Comment:
We need this to be added to all Ring configurations as well, right (to the actual production deployment)?

Reply:
I believe this is resolved, right (in the ADO side)?